resources/views/docs/1/digging-deeper/child-process.md

```diff
@@ -0,0 +1,300 @@
+
+"Spawning" a Child Process is like running a command from the CLI. Any command you can run in the terminal can be a
+Child Process.
+
+**Bear in mind that your Child Process ("command line") arguments may need to differ depending on which platform your
+application is running on (Mac/Linux vs Windows).**
+
+**Where possible, you should explicitly reference binaries by their full path name, unless you can reliably assume that
+the executable you're trying to spawn is available in the user's `PATH`.**
+
+Child Processes are managed by the runtime (Electron/Tauri) but are fully accessible to the Laravel side of your
+application.
+
+---
+
+## Alternatives
+Before deciding to use a Child Process, consider the alternatives available to you. You should pick the most
+appropriate for the problem you're trying to solve:
+
+### Queues
+The [queue runner](queues) is useful for very simply offloading _Laravel_ tasks to the background. Each task must be a
+Laravel queued [Job](https://laravel.com/docs/queues#creating-jobs).
+
+Any queued jobs that don't get processed before your app is quit, will get processed when your application (and the
+queue runner) starts again.
+
+### Scheduler
+The Laravel scheduler runs as normal (every minute) inside a NativePHP application. You can add
+[scheduled tasks](https://laravel.com/docs/scheduling) to your application just as you normally would, to have them run
+on a regular schedule.
+
+Any scheduled tasks that would have run while your application isn't running will be skipped.
+
+**The queue runner and the scheduler are tied to your _application_, not the operating system, so they will
+only be able to run while your application is running.**
+
+### `shell_exec`, `proc_open` etc
+PHP has good built-in support for running arbitrary programs in separate processes. For example:
+
+- [`shell_exec`](https://www.php.net/manual/en/function.shell-exec.php) allows you to run commands and return their
+output to your application.
+- [`proc_open`](https://www.php.net/manual/en/function.proc-open.php) allows you to spin up a command with more control
+over how its input and output streams are handled.
+
+While these can be used in your NativePHP application, consider that they:
+- May block the script that is executing them until the sub-process has finished.
+- May become orphaned from your application, allowing them to continue running after your app has quit.
+
+Runaway orphaned processes could negatively impact your user's system and can become tricky to manage without user
+intervention. You should be cautious about starting processes this way.
+
+---
+
+## Starting a Child Process
+Each Child Process must have a unique alias. This is the name you will use to reference and interact with this process
+throughout your application.
+
+You may start a process using the `ChildProcess` facade:
+
+```php
+use Native\Laravel\Facades\ChildProcess;
+
+ChildProcess::start(
+    cmd: 'tail -n50 storage/logs/laravel.log',
```

Comment:
If I'm not mistaking the `tail` command only stays running when also providing the `-f` flag. Otherwise it will exit with 0

Reply:
added in 3dc6cdae26e95b48610f6dd80bbcd881d5485412